packages/ocular/src/services/organisation.ts

```diff
@@ -151,57 +156,55 @@ class OrganisationService extends TransactionBaseService {
     });
   }
 
-  async updateInstalledApp(
-    app_name: string,
-    data: any
-  ): Promise<AppsArray[] | null> {
+  async updateInstalledApp(app_name: string, data: any): Promise<any | null> {
```

Comment:
Move this function in OAuthService since we are now using Oauth to represent an installed app.

Reply:
Shifted !